main/src/test/java/org/mobilitydata/gtfsvalidator/validator/FeedExpirationDateValidatorTest.java

```diff
@@ -83,16 +83,28 @@ public void feedExpiring7to30DaysFromNowShouldGenerateNotice() {
   }
 
   @Test
-  public void feedExpiring30DaysFromNowShouldGenerateNotice() {
-    assertThat(validateFeedInfo(createFeedInfo(GtfsDate.fromLocalDate(TEST_NOW.plusDays(30)))))
+  public void feedExpiring29DaysFromNowShouldGenerateNotice() {
```

Comment:
Suggestion to modify to feedExpiringAfter30DaysFromNowShouldGenerateNotice instead to conform to the actual guidance from the spec: "If possible, the GTFS dataset should cover at least the next 30 days of service." https://gtfs.org/documentation/schedule/reference/#dataset-publishing-general-practices